### PR TITLE
LRQA-50735 | master

### PR DESCRIPTION
**Pull request by epark18**
https://issues.liferay.com/browse/LRQA-50735

### CODE REVIEW

**Comment by epark18** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
@epark18, can we add more assertions such as description, extension, etc to make sure we cover what is inside the Info Panel?

**Comment by epark18** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase [hunk 1]
```diff
@@ -814,12 +814,53 @@ definition {
...
+		for (var rowEntry : list "Folder1,Document_1.jpeg") {
+			LexiconEntry.selectEntry(rowEntry = "${rowEntry}");
 
+			Click.clickAt(locator1 = "Icon#INFO");
+
+			for (var infoPanel : list "${rowEntry},Primary Key,Description") {
+				AssertElementPresent(
+					locator1 = "DocumentsAndMedia#SIDEBAR_INFO_ACTIVE",
+					value1 = "${infoPanel}");
+			}
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:856]: @epark18 is it possible to assert the actual values of the field as opposed to the field name (primary key/description)?

Comment (epark18) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase:856]: Yes, I should be able to do that by adding individual descriptions for file/folder uploads as well as a specific title for the file upload. Working on the changes now.

**Comment by epark18** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**